Libraries/WebSocket/RCTReconnectingWebSocket.m

```diff
@@ -19,6 +19,27 @@
 
 #if RCT_DEV // Only supported in dev mode
 
+// From https://github.com/apple/swift/blob/ad40c770bfe372f879b530443a3d94761fe258a6/stdlib/public/SDK/os/os_log.m
+typedef struct os_log_pack_s {
+  uint64_t olp_continuous_time;
+  struct timespec olp_wall_time;
+  const void *olp_mh;
+  const void *olp_pc;
+  const char *olp_format;
+  uint8_t olp_data[0];
+} os_log_pack_s, *os_log_pack_t;
+
+static void (*orig__nwlog_pack)(os_log_pack_t pack, os_log_type_t logType);
+
+static void my__nwlog_pack(os_log_pack_t pack, os_log_type_t logType)
+{
+  if (logType == OS_LOG_TYPE_ERROR) {
+    if (strstr(pack->olp_format, "%{public}s %s Connection has no connected handler") == NULL) {
```

Comment:
Since you're comparing the entire string from start to end anyways I suggest changing this check to:
```c
strcmp(pack->olp_format, "%{public}s %s Connection has no connected handler") != 0
```

Reply:
I change it to just compare to `Connection has no connected handler`, the same as in `my_nwlog_legacy_v`